models/users.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.selectUsers = () => {
+  return db.query("SELECT * FROM users").then(({ rows }) => {
```

Comment:
remember your semi-colons in psql statements

Reply:
✅